src/main/java/io/kontur/disasterninja/service/BoundariesService.java

```diff
@@ -11,17 +11,17 @@
 @RequiredArgsConstructor
 public class BoundariesService {
 
-    private static final String BOUNDS = "bounds";
+    private static final String KONTUR_BOUNDARIES = "konturBoundaries";
 
-    private final KcApiClient kcApiClient;
+    private final LayersApiClient layersApiClient;
 
     private final GeometryTransformer geometryTransformer;
 
     public FeatureCollection getBoundaries(GeoJSON geoJSON) {
         Geometry geometry = geometryTransformer.getGeometryFromGeoJson(geoJSON);
         Point point = geometryTransformer.getPointFromGeometry(geometry);
-        List<Feature> featureList = kcApiClient.getCollectionItemsByPoint(point, BOUNDS);
-        return new FeatureCollection(featureList.toArray(Feature[]::new));
+        List<Feature> features = layersApiClient.getCollectionFeatures(point, KONTUR_BOUNDARIES, null);
+        return new FeatureCollection(features == null ? new Feature[]{} : features.toArray(Feature[]::new));
```

Comment:
```suggestion
        return new FeatureCollection(features.toArray(Feature[]::new));
```
`features` cannot be `null`, `getCollectionFeatures` always returns a valid reference. Worst case the list will be empty.